remix/app/components/AppLink/index.tsx

```diff
@@ -1,15 +1,20 @@
 import { Link } from '@remix-run/react';
 
+import { setButtonColor } from '../../helpers/button.helper';
+
 interface ButtonProps {
   name: string;
   href: string;
+  color?: string;
 }
 
-export default function AppLink({ name, href }: ButtonProps) {
+export default function AppLink({ name, href, color }: ButtonProps) {
```

Comment:
Shouldn't this component be called `Button` instead of `AppLink`? There are button props and helpers 😅

Reply:
Oh, that's my bad 🙊, this is my typo on defining the props name, this should be `AppLinkProps` and this is the Link button element that uses for navigation.

Reply:
Improved here [b357dd3](https://github.com/nimblehq/js-framework-benchmark/pull/130/commits/b357dd39fc2364fe1acf62523fa9f7463e88caed)